api/nakadi-event-bus-api.yaml

```diff
@@ -1228,12 +1229,28 @@ definitions:
     properties:
       expected_write_rate:
         type: integer
-      expected_message_size:
+        description: |
+          Write rate for events of this EventType. This rate encompasses all producers of this EventType for
+          a Nakadi cluster.
+
+          Measured in kb/minutes.
+
+      message_size:
         type: integer
-      expected_read_parallelism:
+        description: |
+          Average message size for each Event of this EventType. Includes in the count the whole serialized
```

Comment:
Forget about it, I just checked that one could understand this as been only the contents of the "data" property.